components/balance-actions-handlers.tsx

```diff
@@ -0,0 +1,74 @@
+import { Fragment } from "react";
+import ButtonDialog from "./button-dialog";
+
+export default function BalanceActionsHandlers({
+  info = {
+    title: "",
+    description: "",
+    label: "",
+    caption: "",
+    body: "",
+  },
+  onCloseAction = () => {},
+  onSuccessAction = () => {},
+  disabled = false,
+}: {
+  info: {
+    title: string;
+    description: string;
+    label: string;
+    caption: string;
+    body: string;
+  };
+  onCloseAction: () => void;
+  onSuccessAction: () => void;
+  disabled: boolean;
+}): JSX.Element {
+  return (
+    <ButtonDialog
+      title={info.title}
+      label={info.label}
+      disabled={disabled}
+      className="btn-lg btn-primary w-100"
+      footer={({ hideModal }) => (
+        <>
+          <button
+            className="btn btn-md btn-opac"
+            onClick={() => {
+              onCloseAction();
+              hideModal();
+            }}>
+            Cancel
+          </button>
+          <button
+            className="btn btn-md btn-primary"
+            onClick={() => {
+              onSuccessAction();
+              hideModal();
+            }}>
+            Confirm
+          </button>
+        </>
+      )}>
+      <p className="p-small text-white-50 text-center">{info.caption}</p>
+      <p className="text-center fs-4">
+        {info.body?.split("/").map((sentence: string) => {
```

Comment:
wouldn't it be 10000x easier to just use a children component instead of coming up with a parsing _and_ a templating system _only to create new lines in bold_ ?

Reply:
Gimme a code suggestion please! So I can have a better perception. Btw, this is for "detect" the purple color on the oracles amount text, that has this position changed according of balance type (lock/unlock) :D